src/app/user-and-groups/user-and-groups.page.ts

```diff
@@ -718,6 +718,12 @@ export class UserAndGroupsPage implements OnInit {
   }
 
   goBack() {
+    this.telemetryGeneratorService.generateInteractTelemetry(
```

Comment:
There is one method in TelemetryGeneratorService  to generate back telemetry  add that method.